modules/wallet/persist.go

```diff
@@ -108,12 +90,39 @@ func (w *Wallet) load() error {
 		}
 	}
 
-	// Load the siafunds file, which is intentionally called 'outputs.dat'.
-	var siafundAddresses []types.UnlockHash
-	err = encoding.ReadFile(filepath.Join(w.saveDir, "outputs.dat"), &siafundAddresses)
+	return nil
+}
+
+// load reads the contents of a wallet from a file.
+func (w *Wallet) load() error {
+	var savedKeys []savedKey
+	err := encoding.ReadFile(filepath.Join(persist.HomeFolder, modules.WalletDir, "wallet.dat"), &savedKeys)
```

Comment:
Does it make more sense to read from the home folder as primary or as backup?

What can we assume about people running Sia from multiple locations?

I guess home folder as primary makes sense.